torch/package/_file_structure_representation.py

```diff
@@ -4,27 +4,47 @@
 from .glob_group import GlobPattern, GlobGroup
 
 
-class Folder:
+class Directory:
+    """A file structure representation. Organized as Directory nodes who have lists of
+    their Directory children."""
+
     def __init__(self, name: str, is_dir: bool):
         self.name = name
         self.is_dir = is_dir
-        self.children: Dict[str, Folder] = {}
+        self.children: Dict[str, Directory] = {}
+
+    def get_dir(self, dirs: List[str]):
```

Comment:
```suggestion
    def get_dir(self, dirs: List[str]):
```
Consider adding a return type annotation.

Reply:
And a `Returns:` section to the docstring.